plugins/woocommerce-admin/client/task-lists/fills/products/constants.tsx

```diff
@@ -90,3 +90,6 @@ export const defaultSurfacedProductTypes =
 	onboardingProductTypesToSurfaced.physical;
 
 export const supportedOnboardingProductTypes = [ 'physical', 'downloads' ];
+
+export const SETUP_TASKLIST_PRODUCT_TYPES =
+	'woocommerce_tasklist_experimental_product_types';
```

Comment:
I looked at existing [experimental filters](https://github.com/woocommerce/woocommerce/blob/2f77de3f185644c0ef3b9b9cb5cdae6f9ca7b31e/plugins/woocommerce-admin/client/analytics/report/variations/table.js#L297), I think this is a better name
```suggestion
	'experimental_woocommerce_tasklist_product_types';
```